subsys/caf/modules/caf_shell.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <ctype.h>
+#include <stdlib.h>
+#include <zephyr/shell/shell.h>
+#include <app_event_manager.h>
+#include <caf/events/button_event.h>
+
+static const char set_button_cmd_help_str[] =
+	"Submit a button_event with user-defined key ID and pressed state\n"
+	"  Key ID           Single-digit numeric ID of the button\n"
+	"  Pressed state    'y' or 'n'\n";
+
+static int set_button_handler(const struct shell *shell, size_t argc, char **argv)
+{
+	if ((strlen(argv[1]) > 1) || !isdigit(*argv[1])) {
```

Comment:
I've been thinking about bigger key IDs, in case someone has a board with 20 LEDs. Using decimal would be easier to remember than hex (for me at least)